frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -59,6 +60,8 @@ const TranslationPage = () => {
   // Font Size Slider
   const [fontSize, setFontSize] = useState<string>("12px");
 
+  const [stage, setStage] = useState<string>("");
```

Comment:
nit: move these lines to 43, just before the "AutoSave" comment and after the "Story Data" comment